docs/source/VUDRM-token.md

```diff
@@ -0,0 +1,253 @@
+# VUDRM TOKEN 
+
+The VUDRM token has two purposes: authentication and the delivery of the DRM policy to the license server. It also represents a signed authorisation on the client's behalf for Vualto to issue a DRM license to the holder of the token, issuing a VUDRM token to a player will grant that player access to the DRM-protected content.
+
+Due to the first purpose VUDRM tokens have a limited lifetime and are designed to be single use. Please contact support@vualto.com if you do not know what the VUDRM token's TTL is for your account.
+
+The second purpose allows the user's playback rights for individual pieces of content to be set dynamically. A single user may be granted different rights on a single piece of content depending on business requirements.
+
+VUDRM tokens should be generated using the [VUDRM token API](vudrm-token-api). The request to the [VUDRM token API](vudrm-token-api) should be made from a server side application and the VUDRM token should then be delivered to the client side for use by a player in a license request.
```

Comment:
Not sure linking like this will work once RTD picks this up, I believe `/projects/vudrm/en/latest/VUDRM-token.html#vudrm-token-api` should definitely work but there may be a shorter way, a small amount of testing would need to be done.